internal/ansible/proxy/requestfactory/requestinfo.go

```diff
@@ -127,7 +127,7 @@ type RequestInfoFactory struct {
 // /api
 // /healthz
 
-func (r *RequestInfoFactory) NewRequestInfo(req *http.Request) (*RequestInfo, error) {
+func (r *RequestInfoFactory) NewRequestInfo(req *http.Request) (*RequestInfo, error) { //nolint:gocyclo
```

Comment:
IHMO here we need to add TODO: Try to reduce the complexity of this and remove the // nolint:gocyclo.